contentcuration/contentcuration/frontend/channelEdit/mutations.js

```diff
@@ -0,0 +1,7 @@
+export function SET_VIEW_MODE(state, viewMode) {
+  state.viewMode = viewMode;
+}
+
+export function SET_VIEW_MODE_OVERRIDES(state, overrides) {
```

Comment:
One thing to note- with the current Studio interface, this is also attached to the sessionStorage so it maintains their preferred state

Reply:
What do you mean by "current Studio interface"? This still retains their preferred state, if that's not clear, let me know.

Reply:
The one that's deployed to studio.leq.  Ah cool! I see the persistFactory now. Feel free to disregard